src/logic/addressBook/store/middleware/addressBookMiddleware.js

```diff
@@ -31,8 +31,11 @@ const addressBookMiddleware = (store: Store<GlobalState>) => (next: Function) =>
 
     switch (action.type) {
       case ADD_ENTRY: {
-        const notification = getNotificationsFromTxType(TX_NOTIFICATION_TYPES.ADDRESSBOOK_NEW_ENTRY)
-        dispatch(enqueueSnackbar(enhanceSnackbarForAction(notification.afterExecution.noMoreConfirmationsNeeded)))
+        const { isOwner } = action.payload.entry
+        if (!isOwner) {
```

Comment:
could you please explain why we check if the entry is not an owner?

Reply:
Yes, because if the entry is an owner, that means that's an entry added by the user manually, in that case, we should display the notification. Otherwise, when the addressbook loads, it will display notifications for the owner's entrys all the time